snuba/datasets/entities/outcomes.py

```diff
@@ -0,0 +1,99 @@
+from datetime import timedelta
```

Comment:
Trivial question regarding organization -- will we ever have an `event.py` in one dataset that is not the same `event.py` in another dataset? If the answer is yes, should we have the package be `snuba/datasets/<dataset>/entities/<blah>.py` ?   

If the answer is no, should the entities be hoisted outside of datasets? Or even if yes, `snuba/entities/<dataset>/<blah>.py` ?

Reply:
I am mirroring the way storages are organized. I don't know of any use case where we would want two different versions of the transactions entity. The utility of this right now is that the transaction entity can be used by both the Discover and Transaction dataset.